src/Not/Not/Authentication/User/NUserResolution.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+
+namespace Not.Authentication.User;
+
+public class NUserResolution : IUserResolver
```

Comment:
Implementations should be named after the implemented interfaces - `NUserResolver`